browser/src/unocommands.js

```diff
@@ -291,7 +293,7 @@ var unoCommandsArray = {
 	'InsertSlideTitleField':{presentation:{menu:_('Slide Tit~le'),},},
 	'InsertSlidesField':{presentation:{menu:_('Slide ~Count'),},},
 	'InsertSoftHyphen':{global:{menu:_('Insert s~oft Hyphen'),},},
-	'InsertSparkline':{spreadsheet:{menu:_('Insert Sparkline...'),},},
+	'InsertSparkline':{spreadsheet:{menu:_('Sparkline...'),},},
```

Comment:
shouldn't be changed

Reply:
It's fine, the string was changed in core.
```
commit 273ebcc95defc82a9ec4073f8c0a951a9da89707
Author: Julien Nabet <serval2412@yahoo.fr>
Date:   Mon Jul 11 11:21:56 2022 +0200

    tdf#149738: Minor inconsistencies in sparkline related UI strings
    
    Change-Id: I5b4c31e69743ad519a8bcc99fa3a5a7512a4cc18
    Reviewed-on: https://gerrit.libreoffice.org/c/core/+/136955
```